packages/d/drm-info/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : drm_info
```

Comment:
The name of the directory is `drm-info`, but the package name is `drm_info`. Which one should it be?

Reply:
Gotask complained that _ was not ask ascii so I made the underscore a dash.

Reply:
`Package names are restricted to US ASCII lowercase letters, numbers, dashes, and, dots.`

Reply:
Per discussion on Matrix, it used to be a tooling limitation that we couldn't use underscores in package names. Presumably, this is no longer the case, since there are packages in the repository that have underscores in their name.

The main thing is that the folder name and package name match, whichever way it goes. While I think that using an underscore is not nearly as intuitive to type as a dash when using a package manager, the name of the upstream project is with an underscore, and both Fedora and the AUR also use the underscore. So, I would say to change the folder to be `drm_info`.

Reply:
Got it, then lm_sensors needs to be changed as well